plugins/kubernetes/test/controllers/kubernetes_dashboard_controller_test.rb

```diff
@@ -36,30 +44,112 @@
 
   private
 
-  def api_response
-    response = JSON.parse(parse_json_response_file('kubernetes_pod_api_response'))
```

Comment:
I think I removed them in the branch that is focused solely in the tests. In this PR I just tried to fix the tests that were failing.